src/identifiers/logic.py

```diff
@@ -344,6 +341,49 @@ def render_doi_from_pattern(article):
     return '{0}/{1}'.format(doi_prefix, doi_suffix)
 
 
+def preview_registration_information(article):
+    """
+    Generates a rudimentary printout of metadata
+    for proofing by the end user before attempting
+    to register the DOI with Crossref.
+    """
+    if article.journal.use_crossref:
+        crossref_context = create_crossref_context(article)
+
+        # FrozenAuthors do not exist yet
+        # crossref_context['authors'] = article.authors.all()
+
+        exclude = ['batch_id', 'timestamp', 'depositor_email', 'registrant',
+                   'now', 'timestamp_suffix', 'print_issn', 'citation_list']
+        for k in exclude:
+            crossref_context.pop(k)
+
+        metadata_printout = 'We will try to send the following ' \
+                            'provisional metadata to Crossref:<br><br>'
+
+        for k, v in crossref_context.items():
+            if k == 'authors':
+                i = 0
+                for author in crossref_context['authors']:
+                    i += 1
+                    for prop in ['first_name', 'middle_name', 'last_name',
+                                 'department', 'institution', 'orcid']:
+                        if hasattr(author, prop) and getattr(author, prop) != None:
+                            val = getattr(author, prop)
+                        else:
+                            val = ''
+                        metadata_printout += f'author{str(i)}_{prop}: {val}<br>'
+            else:
+                metadata_printout += f'{k}: {"" if v == None else v}<br>'
+        metadata_printout += '<br>If you update metadata after acceptance, ' \
+                             'we will resend it to Crossref on publication.'
```

Comment:
I revised the template to highlight whether use_crossref is on and explain the timing of DOI deposits, so that it can be independent of registration_preview.